## Test

Should fail since y has two definitions when x = 0

```catala
declaration scope A:
  internal x content integer
  output y content integer

scope A:
  definition x under condition (6*7 = 42) and (false or (true and 1458 / 27 = 54.))
  consequence equals 0

  definition y under condition x <= 0 consequence equals -1
  definition y under condition x >= 0 consequence equals 1
  
```
```catala-test-inline
$ catala Proof --disable_counterexamples 
[WARNING] [A.y] At least two exceptions overlap for this variable:
┌─⯈ tests/test_proof/bad/no_vars-conflict.catala_en:8.10-8.11:
└─┐
8 │   output y content integer
  │          ‾
  └─ Test
Counterexample generation is disabled so none was generated.
```
